docs/markdown/connecting-to-testnet.md

```diff
@@ -0,0 +1,61 @@
+# Connecting to an ILP Testnet
+
+This tutorial will help you start [`moneyd`](https://github.com/interledgerjs/moneyd) locally on your computer and 
+connect it to an [ILP router](https://github.com/interledgerjs/ilp-connector) on the testnet.
+
+This tutorial describes how to:
+
+1. Configure and start `moneyd` to connect to an ILP router on the testnet
+2. Send value to a [payment pointer](https://interledger.org/rfcs/0026-payment-pointers/) on the testnet
+
+## Before you begin
+For this tutorial, you will need `moneyd` and SPSP installed on your computer. You can learn how to install `moneyd` 
+and SPSP from the [Getting Started](getting-started.md) tutorial.
+
+NOTE: For this tutorial you do not need to use real cryptocurrency. Since you will be sending value through an 
```

Comment:
I think it should be spelled out explicitly that in this tutorial, testnet _XRP_ is the asset used to send and receive money with the peered connector.

Reply:
I have provided a link to the XRP testnet faucet. 
https://github.com/interledger/interledger.github.io/pull/108/commits/06659009ab81d44db7db3df5f318b5bae704a4ba#diff-95b449acc2e692458b91372a5b6e33e7R21